Source/EasyNetQ/DefaultSerializer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Reflection;
+
+namespace EasyNetQ
+{
+    internal sealed class DefaultSerializer : ISerializer
+    {
+        private static readonly Type _jsonBackwardCompatibilitySerializerType;
+        private readonly ISerializer _jsonBackwardCompatibilitySerializer;
+
+        static DefaultSerializer()
+        {
+            try
+            {
+                var asm = Assembly.Load(new AssemblyName("EasyNetQ.Serialization.NewtonsoftJson"));
```

Comment:
Is it worth to check the existence of the type and catch an error in `static constructor`, but after it throw the exception in `constructor` based on a null value of the field?

Reply:
@sungam3r What do you think about it?